static/js/message_edit.js

```diff
@@ -178,6 +179,17 @@ exports.show_topic_edit_spinner = function (row) {
     $(".topic_edit_cancel").hide();
 };
 
+exports.show_topic_move_spinner = function (editing) {
+    const spinner = $("#move_topic_modal .topic_move_spinner");
+    if (editing) {
+        loading.make_indicator(spinner);
+        $("#move_topic_modal .modal-footer").hide();
+    } else {
+        loading.make_indicator(spinner);
+        $("#move_topic_modal .modal-footer").show();
+    }
+};
```

Comment:
I usually find it cleaner to make separate show/hide methods just for readability, even if they share a line of code.